### PR TITLE
Delete asset button

### LINKED ISSUE #4291: Add delete button in asset detail page
**Is your feature request related to a problem? Please describe.**

Currently, we are not able to delete a created asset on the asset page

**Describe the solution you'd like**

Add a delete button in the asset detail page, and only district admin and above should be able to do it

image

### PR DESCRIPTION
**Pull request by skks1212**
## Proposed Changes

- Fixes #4291 

image
(Note - Export button will be moved later on)

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by nihal467** [comment]
@skks1212 the Ui is not responsive, make it responsive

https://deploy-preview-4304--care-net.netlify.app/facility/eeb23cb7-1524-479a-825e-660311386b47/assets/5c088496-f800-4bf7-9ac6-02f60dde3a4c

image

**Comment by nihal467** [comment]
@skks1212 https://deploy-preview-4304--care-net.netlify.app/facility/fdad4400-bd11-49dd-9da1-5be0e79fa14e/assets/4ba8c734-a30c-42cc-93c6-b5e5b3028c28

image


the responsiveness is still not fixed, and check the cypress too

**Comment by nihal467** [comment]
@khavinshankar @gigincg  conditionally approving this PR, as it serves the functionality, and the responsiveness issue will be followed up with #4379 

@skks1212 check the cypress failed

File: src/Components/Assets/AssetManage.tsx [hunk 1]
```diff
@@ -175,6 +189,18 @@ const AssetManage = (props: AssetManageProps) => {
     if (asset) downloadJSON(asset);
   };
 
+  const handleDelete = async () => {
+    const confirm = window.confirm(
+      "Are you sure you want to delete this asset?"
+    );
```
Comment (khavinshankar) [inline src/Components/Assets/AssetManage.tsx:195]: can we have a custom modal for this instead of just window.confirm?

**Review by khavinshankar — APPROVED** [review]

File: src/Components/Assets/AssetManage.tsx (hunk 1 above)
Comment (khavinshankar) [reply src/Components/Assets/AssetManage.tsx:195]: create a new issue regarding this #4434

**Pull request merged**